wranglertools/import_data.py

```diff
@@ -139,6 +140,26 @@ def getArgs():  # pragma: no cover
 ]
 
 
+ALLOWED_MIMES = (
+    'application/pdf',
+    'application/zip',
+    'text/plain',
+    'text/tab-separated-values',
+    'text/html',
+    'application/msword',
+    'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+    'application/vnd.ms-excel',
+    'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+    'image/png',
+    'image/jpeg',
+    'image/gif',
+    'image/tiff',
+)
+
+# If modifying these scopes, delete the file token.json.
```

Comment:
Where would this be located if installing from PyPi? Make targets fixing these things may be useful... You may also want to read the `SCOPES` from a config file.